### DISCUSSION TITLE
Now playing integration / dll

### DISCUSSION THREAD

**Discussion by EnderSpy29** [KRTirtho/spotube#565] [Q&A] [answered]
So is there a .dll or something a program can acess to see what spotube is playing? and to control the player

**Comment by KRTirtho** [comment, marked as answer]
Sorry. Spotube doesn't expose such API. Although, it uses [SMTC](https://learn.microsoft.com/en-us/uwp/api/windows.media.systemmediatransportcontrols?view=winrt-22621) for windows media control. But I don't know if there's any Windows API that allows to read currently playing media

**Reply by KRTirtho** [reply]
Here's a [Stackoverflow question](https://stackoverflow.com/questions/65011660/how-can-i-get-the-title-of-the-currently-playing-media-in-windows-10-with-python) that might help you

**Reply by EnderSpy29** [reply]
ok cool! i think with a bit of shenanigans i should be able to integrate it into my rainmeter setup. (the things we do lol)

**Reply by KRTirtho** [reply]
I also used to tinker windows a lot before moving to linux :joy:
